breadStore/main.js

```diff
@@ -1,42 +1,19 @@
-function breadlist() {
-  fetch("products.json")
-    .then((response) => response.json())
-    .then((data) => {
-      const articleList = document.getElementById("cards");
-      const breadsList = data.breads;
-      for (let i = 0; i < breadsList.length; i++) {
-        const bread = breadsList[i];
+import { breads } from "./products.js";
 
-        const article = document.createElement("article");
-        article.classList.add("section__card");
+const articleList = document.querySelector("#cards");
 
-        const name = document.createElement("h3");
-        name.textContent = bread.name;
-        article.appendChild(name);
+const breadsList = breads.map((bread) => {
+  const hashtags = bread.hashtags
+    .map((tag) => `<li class="section__tag-list">${tag}</li>`)
+    .join(" ");
+  return `
+    <article class="section__card">
+      <h3>${bread.name}</h3>
+      <ul class="section__tag">${hashtags} </ul>
+      <img class="section__img" src="./imgs/${bread.src}"/>
+      <button class="section__like" id="btn${bread.id}" type="button">♥</button>
+    </article>
+  `;
+});
```

Comment:
```suggestion
const breadsList = breads.map(({name, src, id, hashtags }) =>  `
    <article class="section__card">
      <h3>${name}</h3>
      <ul class="section__tag">
      ${hashtags
        .map((tag) => `<li class="section__tag-list">${tag}</li>`)
         .join(" ")} 
         </ul>
      <img class="section__img" src="./imgs/${src}"/>
      <button class="section__like" id="btn${id}" type="button">♥</button>
    </article>
  `;
);
```
이렇게도 구조분해 할당해서 쓸 수 있을고야!